app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,5 @@
 <resources>
     <string name="app_name">MinuteJournal</string>
-<<<<<<< HEAD
```

Comment:
This looks like the result of a merge conflict that was not properly merged.  Best to clean these up, because they render the XML file invalid... which will cause build issues.